### PR TITLE
V2 amends joint replacment

### PR DESCRIPTION
**Pull request by bottlerocket**
Added new pages for Tier 2 content.

### CODE REVIEW

**Comment by marcofucci** [comment]
@teneightfive might be better than me at reviewing this

**Comment by teneightfive** [comment]
@tim-allan images have been export at 1x but need to be cut out at 2x and then they are scaled to 1x so the text looks sharp and readable on retina which is what we'll be testing on I believe.

File: assets/stylesheets/main.css [hunk 1]
```diff
@@ -17,4 +17,5 @@ img {
 
 a {
   position: absolute;
+  /*border:1px solid red;*/
```
Comment (teneightfive) [inline assets/stylesheets/main.css]: if this isn't needed, just delete it rather than comment out

File: hernia--steps-to-surgery/cover.html [hunk 2]
```diff
@@ -19,17 +19,44 @@
...
+                height: 30px;"></a>
+
+
+      <a href="see-the-surgeon.html"
+         style="top: 1158px;
+                left: 105px;
+                width: 250px;
+                height: 30px;"></a>
+
+      <!-- Checking you are fit for surgey --> checking-you-are-fit
```
Comment (teneightfive) [inline hernia--steps-to-surgery/cover.html]: this text is outside the comment and currently shows up at the bottom of the document

**Pull request merged**